pkgs/top-level/all-packages.nix

```
@@ -26814,4 +26814,8 @@ in
   vpsfree-client = callPackage ../tools/virtualization/vpsfree-client {};
 
   gpio-utils = callPackage ../os-specific/linux/kernel/gpio-utils.nix { };
+
+  html5validator = callPackage ../applications/misc/html5validator {
+    inherit (python36Packages) buildPythonPackage nose;
+  };
```

use python package scope
```suggestion
  html5validator = python36Packages.callPackage ../applications/misc/html5validator { };
```